src/JSUtils/ScopeManager.js

```diff
@@ -89,7 +91,7 @@ define(function (require, exports, module) {
      * Read in the json files that have type information for the builtins, dom,etc
      */
     function initTernEnv() {
-        var path = ExtensionUtils.getModulePath(module, "node/node_modules/tern/defs/"),
+        var path = [_absoluteModulePath, "node/node_modules/tern/defs/"].join("/"),
```

Comment:
This needs to be unit tested both on Windows, MAC and Linux.

Reply:
@navch Did you get a chance to unit test this PR on all the platforms? Please let me know once unit testing is done, so that I can go ahead with merging the PR.

Reply:
I tested this. Its working fine. :) Path will be like "C:/usr/brackets/src/JSUtils/./node/TernNodeDomain"

Do you want me to remove this "./". Bdw its working fine.